### DISCUSSION TITLE
Multiple Zigbee2MQTT feeding into single Mosquitto

### DISCUSSION THREAD

**Discussion by Andrewkha** [Koenkk/zigbee2mqtt#20938] [Devices]
Hello!
I have two apartments connected into the single network via VPN. And my plan to monitor/automate both locations with the single HomeAssistant instance. My initial setup:

Location 1: QNAP NAS with HA container, Mosquitto MQTT container
Location 2: Ubuntu miniPC with Zigbee2MQTT container,

I put several Zigbee devices into location 2, registered them into local Zigbee2MQTT instance and configured to use Mosquitto server in Location 1, which in turn I registered with HA. Everything worked fine.

Later I decided to add some automation into Location 1. I bought another Sonoff adapter, inserted into my NAS and added another container with Zigbee2MQTT on NAS. And use the existing mosquitto instance. Everything worked fine at the fisrt glance. However later I discovered that smart devices from location 1 disappeared from the HA. If I shut the zigbee2mqqt in Location 2, they get back. So looks like there's a conflict with two different zigbee2mqtt instances using the single mosquitto instance. 

I could create another mosquitto instance but the problem is that HA can use only 1 MQTT integration. Is there any solution for this problem?

**Comment by Andrewkha** [comment]
My bad... I shoud have RTFM more carefully before posting this

**Reply by jasonhollis** [reply]
When manual are you referring to because I can't seem to find it.  All I have found is:  In case you setup multiple instances of Zigbee2MQTT it's important to use a different base_topic and channel. This can be configured in the [configuration.yaml](https://www.zigbee2mqtt.io/guide/configuration/).  Which does not tell me much about how to get it done.  I guess I'm slow but it would be nice to see an example config.
